### DISCUSSION TITLE
Unable to build image successfully

### DISCUSSION THREAD

**Discussion by presIey** [DustinBrett/daedalOS#418] [General]
Hello, I cannot build the docker image smoothly. Could you package the image and push it to dockerhub?

**Comment by DustinBrett** [comment]
Could you clarify the issue you are having? I am able to get it going with only these 2 commands on a fresh repo.

https://github.com/DustinBrett/daedalOS?tab=readme-ov-file#docker

```
docker build -t daedalos .
docker run -dp 3000:3000 --rm --name daedalos daedalos
```
